db/migrate/20230427195343_create_attendee_to_events.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAttendeeToEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :attendee_to_events do |t|
```

Comment:
In Rails the name convention for these join tables is `<entity 1><entity 2 in plural>`, so in this case it should be either `event_attendees` or `attendee_events`. `event_attendees` makes more sense in this case, in english.

Reply:
thanks! done!